#geometry_msgs/Transform target_pose
geometry_msgs/Vector3 target_pos
geometry_msgs/Vector3 target_rot
bool orientation
bool full_body
bool debug
---
bool status
